test/auth/test_credentials.py

```diff
@@ -6,194 +6,178 @@
 from cartoframes.auth import Credentials
 from cartoframes.auth.credentials import _USER_CONFIG_DIR, _DEFAULT_PATH
 
+# FIXME python 2.7 compatibility
```

Comment:
We can remove this FIXME note

Reply:
I think  it will be useful when we remove the Python 2 support